### DISCUSSION TITLE
Adjusting context lines in diff view?

### DISCUSSION THREAD

**Discussion by MasterInQuestion** [community/community#122412] [Repositories]
Is there any way to specify the desired number of context lines of diff view?
    (e.g. via "c=100" alike request parameter; corresponding to [&#x60;diff&#x60;](https://www.gnu.org/software/diffutils/manual/diffutils.html#diff-Options) "-C")

    Related:
    https://github.blog/2013-12-02-expanding-context-in-diffs/

    ----

    Notable use case:
    [ https://github.com/MasterInQuestion/attach/commit/e18e5cd26b9d27bacefa685ba70d0d41bc7d7cbc?c=1000](https://github.com/MasterInQuestion/attach/commit/e18e5cd26b9d27bacefa685ba70d0d41bc7d7cbc?c=1000)
    (alike)
    https://trac.ffmpeg.org/wiki/colorspace?action=diff&contextlines=8&version=31
    https://trac.ffmpeg.org/wiki/Scaling?action=diff&contextlines=5&old_version=30&version=34

**Comment by bradlitterell** [comment]
This should be a default profile setting that I can set for how I want to see PR diffs.
